app/domain/logs.rb

```diff
@@ -389,6 +389,11 @@ module AuthnJwt
         msg: "Fetching jwks-uri signing key",
         code: "CONJ00076D"
       )
+
+      FetchingTokenString = ::Util::TrackableLogMessageClass.new(
+        msg: "Fetching jwt value from request",
```

Comment:
@shulifink  please review

Reply:
@hilagross ^

Reply:
maybe 
```suggestion
        msg: "Fetching JWT token from request",
```

??

Reply:
ie... to be consistent with the above...